bin/monitor/tui_jobs_display.ml

```diff
@@ -0,0 +1,142 @@
+open Import
+open Dune_tui.Import
+module Job = Dune_rpc.Job
+
+type state = Rpc_running_jobs.State.t
+
+let number_of_jobs = Lwd.var 0
+
+let render (state : Rpc_running_jobs.State.t) =
+  let now = Unix.gettimeofday () in
+  let jobs =
+    Rpc_running_jobs.Job_map.to_list_map
+      state.displayed_jobs_by_id
+      ~f:(fun (_ : Job.Id.t) displayed_job -> displayed_job.row, displayed_job)
+    |> Int.Map.of_list_exn
+  in
+  (* CR-someday amokhov: We should highlight jobs that were running for longer than 5s
```

Comment:
Yes but they are relevant.

Reply:
They are duplicated because we have the same issue with both jobs display backends. The coloring issues will have to be solved independently however the description shortening and anonymous action filtering could share a solution.

Reply:
Why do coloring issues have to be solved independently? We can use `Pp.t` to set the colors in both.